portfolio/src/main/webapp/style.css

```
@@ -6,6 +6,9 @@
   display: flex;
   flex-direction: row;
 }
+.prim_color{
```

[4.1.4](https://google.github.io/styleguide/htmlcssguide.html#ID_and_Class_Name_Delimiters): Separate words in class names by a hyphen.
```suggestion
.prim-color {
```